script/install_tbb.sh

```diff
@@ -20,7 +20,6 @@ then
     travis_retry sudo apt-get -y --quiet --allow-unauthenticated --no-install-recommends install libtbb-dev
 elif [ "$ALPAKA_CI_OS_NAME" = "macOS" ]
 then
-    brew unlink python@2
```

Comment:
Github seems to have removed python 2 from the MacOS images so we can not (and do not need to) remove it manually.

Reply:
I opened the same fix in #1143. I closed now #1143 because this PR passed already the CI.
Please do not hide those fixes together with features.

I added the label `bug` and extent the description of this PR.